kratos/processes/levelset_convection_process.h

```diff
@@ -249,13 +329,22 @@ class LevelSetConvectionProcess
         for (int i_node = 0; i_node < static_cast<int>(mpDistanceModelPart->NumberOfNodes()); ++i_node){
             auto it_node = mpDistanceModelPart->NodesBegin() + i_node;
             it_node->FastGetSolutionStepValue(VELOCITY) = mVelocity[i_node];
-            it_node->FastGetSolutionStepValue(VELOCITY,1) = mVelocityOld[i_node];
+#ifdef KRATOS_DEBUG
+            KRATOS_ERROR_IF(dt_factor < 1.0e-12)
+                << "ERROR: dt_factor shoild be larger than zero." <<std::endl;
+#endif
```

Comment:
Why only in debug mode?

Reply:
I don't know :)

Reply:
I'd check it always (this is a cheap one)